observability/network_observability/configuring-operator.adoc

```diff
@@ -6,28 +6,29 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update the Flow Collector API resource to configure the Network Observability Operator and its managed components. The  Flow Collector is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`.
+You can update the `FlowCollector` API resource to configure the Network Observability Operator and its managed components. The  `FlowCollector` is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`. For more information, see the xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[FlowCollector API reference].
```

Comment:
Nit - the section that this link leads to technically uses "specifications" instead of reference. If this is intended to point to the specs sections, I'd update the link text:
```suggestion
You can update the `FlowCollector` API resource to configure the Network Observability Operator and its managed components. The  `FlowCollector` is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`. For more information, see the xref:../../observability/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[FlowCollector API specifications].
```

If the link is intended to point to the overall assembly I'd update the link text _and_ the anchor:
```suggestion
You can update the `FlowCollector` API resource to configure the Network Observability Operator and its managed components. The  `FlowCollector` is explicitly created during installation. Since this resource operates cluster-wide, only a single `FlowCollector` is allowed, and it has to be named `cluster`. For more information, see the xref:../../observability/network_observability/flowcollector-api.adoc#flowcollector-api[FlowCollector configuration parameters].
```

In either case the "FlowCollector API reference" phrasing is only used in the topic map name for this page/file, should that match the assembly title?

Reply:
This is a great nit to catch. You're right, there's a little mis-matching in the names with the topic-map and assembly. Out of these suggestions, I think I will change the assembly title to "FlowCollector API reference".